### PR TITLE
edit yt category client side

### PR DESCRIPTION
**Pull request by akash1810**
not linked to datastore yet as api needs updating, but this will be done in a different PR

yt-cat

### CODE REVIEW

File: public/video-ui/src/actions/YoutubeActions/getCategories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+function receiveCategories(categories) {
+  return {
+    type: 'YT_CATEGORIES_GET_RECEIVE',
+    categories: categories,
+    receivedAt: Date.now()
+  };
+}
+
```
Comment (ShaunYearStrong) [inline public/video-ui/src/actions/YoutubeActions/getCategories.js]: We should add something for errors here, as in the other actions.

File: public/video-ui/src/components/VideoEdit/formComponents/YoutubeCategory.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+class YoutubeCategorySelect extends React.Component {
+
+  hasCategories () {
```
Comment (ShaunYearStrong) [inline public/video-ui/src/components/VideoEdit/formComponents/YoutubeCategory.js]: doesn't this need arrow function syntax to bind to `this` correctly?

File: public/video-ui/src/actions/YoutubeActions/getCategories.js (hunk 1 above)
Comment (akash1810) [inline public/video-ui/src/actions/YoutubeActions/getCategories.js]: Done.

File: public/video-ui/src/components/VideoEdit/formComponents/YoutubeCategory.js (hunk 2 above)
Comment (akash1810) [inline public/video-ui/src/components/VideoEdit/formComponents/YoutubeCategory.js]: Fixed now.

**Pull request merged**